tools/pot/tests/test_graph_with_stats.py

```diff
@@ -17,7 +17,8 @@
 TEST_MODELS = [
     ('resnet_example', 'pytorch', 'DefaultQuantization', 'CPU'),
     ('resnet_example', 'pytorch', 'RangeSupervision', 'ANY'),
-    ('act_act_example', 'pytorch', 'DefaultQuantization', 'GNA')
+    ('act_act_example', 'pytorch', 'DefaultQuantization', 'GNA'),
+    ('concat_depthwise_model', 'pytorch', 'MinMaxQuantization', 'CPU'),
```

Comment:
xml ref file contains fq shapes, so it's checking that FQ before concat are perchannel

Reply:
Good! So, adding in the test_unify_scales.py is not needed anymore?

Reply:
`test_graph_with_stats` is calling `find_unify_scales` in
1) `get_configuration_by_preset`, where unified quantizers affects fq configuration, which is used for activation statistics layout computation
2) `compute_statistics_layouts`, where unified quantizers affects fq configuration, which is used for filling fq parammeters
3) `unify_fq_scales` to actually unify computed parameters of FQs

And in `test_unify_scales` modified function `find_unify_scales` is called explicitly, and output values is compared with reference values explicitly. This test is more like an unit test, and I suggest to keep it as well. It shouldn't increase testing time in any significant extent